docs/develop/howtos/custom_code.md

```diff
@@ -181,12 +171,12 @@ ReactDOM.render(
 );
 ```
 
-Now let's register this file in the webpack configuration. In the file `webpack.py`, add the definition of a new webpack bundle, with all the JavaScript files that it will contain (in this case, only one):
+Now let's register this file in the Webpack bundle. In the file `webpack.py`, add a new entry, pointing to your new JavaScript file:
 
-```python
+```python hl_lines="10"
 from invenio_assets.webpack import WebpackThemeBundle
 
-support = WebpackThemeBundle(
+theme = WebpackThemeBundle(
```

Comment:
I'm reusing the [default Webpack bundle](https://github.com/inveniosoftware/cookiecutter-invenio-rdm/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/site/%7B%7Bcookiecutter.package_name%7D%7D/webpack.py), since it's also already included in the `setup.cfg`. This might make the process less error-prone and doesn't require re-running `pipenv run pip install -e .`

Reply:
Ok then @alejandromumo was right, no need to add the entry point... I forgot that we had one already.